SITCOM-1397_Extract_dominant_frequencies_from_sound_waves_in_video_associated_with_vibrations_on_Top-End_Assembly_II.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
```

Comment:
<p>In my case, this cell is taking a very long time to execute. I stopped it after 36 minutes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
This cell is deleted, it doesn't make any changes in the result.